proxy/ProxyTransaction.cc

```diff
@@ -221,6 +221,24 @@ ProxyTransaction::has_request_body(int64_t request_content_length, bool is_chunk
   return request_content_length > 0 || is_chunked;
 }
 
+bool
+ProxyTransaction::is_read_closed() const
```

Comment:
Is this used in some subsequent PRs? Seems a little odd to have this "no-op" method, which is never called as far as I can tell.

Reply:
Probably leftover from the H2 to origin pull back.

Reply:
Removed for now.  Will be added back with the H2 to origin logic.